lib/filter/enemy.js

```diff
@@ -99,7 +99,31 @@ exports.update = function (json) {
       }
     }
     */
-
+   
+		console.log( 'Enemy Drops:');
+		var drops = 0;
+		for (enemy in json[round].enemy) {
+			if (json[round].enemy[enemy].children[part].drop_item_list.length > 0) {
+				drops = 1;
+				for (var item in json[round].enemy[enemy].children[part].drop_item_list) {
```

Comment:
Ops, my bad, didn't notice that it was about the items (I'm still not used with github's interface). I haven't really checked enemy drops when I was doing it, I remember normal enemies would drop on their own entity, but I never really checked for bosses, but if you fix the whole thing on bosses (Taking out the second loop), be sure that the drop is in the first entity, or else you won't see it.